iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalCatalog.java

```diff
@@ -50,11 +58,12 @@ public class OpenHouseInternalCatalog extends BaseMetastoreCatalog {
   protected TableOperations newTableOps(TableIdentifier tableIdentifier) {
     return new OpenHouseInternalTableOperations(
         houseTableRepository,
-        fileIOManager.getFileIO(storageManager.getDefaultStorage().getType()),
+        getFileIO(tableIdentifier),
         snapshotInspector,
         houseTableMapper,
         tableIdentifier,
```

Comment:
lets not keep FileIOManager as the last parameters, I've observed generally that metrics related params are kept last

Reply:
done